### ISSUE TITLE
Updating existing ownerReferences apiVersion

### ISSUE THREAD

**Issue by sebgl** [kubernetes-sigs/kube-storage-version-migrator#49]
This is more a question than a real project issue, I'm sorry if this is not the right place for it (happy to move it elsewhere).

I was reading through this project to do something similar for my own CRDs. The underlying goal being to eventually remove support for eg. CRD v1 when we release CRD V3.

I understand how existing resources can be updated to use the latest storage version.
What happens with other resources that have an `ownerReference` set to the resource whose apiVersion was updated?

I could have eg. a `Secret` with an existing `ownerReference` pointing to an instance of my CRD with `apiVersion: v1`. Would that cause an issue if I remove the CRD V1 at some point?

Should I include "migrate all existing ownerReferences" as part of "migrate all existing resource storage versions"?

**Comment by sebgl** [comment]
Closing following this [Slack convo](https://kubernetes.slack.com/archives/C0EG7JC6T/p1575039839348100).

tl;dr: deprecated `apiVersion` in `ownerReferences` don't matter much, they are not use to GC resources.
